packages/plugins/contracts/foundry/Sphinx.sol

```diff
@@ -448,21 +448,30 @@ abstract contract Sphinx {
                 manager.setStorage{ gas: bufferedGasLimit }(rawActions, _proofs);
             } else {
                 vm.recordLogs();
-                manager.executeInitialActions{ gas: bufferedGasLimit }(rawActions, _proofs);
+                // manager.executeInitialActions{ gas: bufferedGasLimit }(rawActions, _proofs);
+                (bool success, bytes memory result) = address(manager).call{ gas: bufferedGasLimit }(abi.encodeWithSignature("executeInitialActions((uint8,uint256,bytes)[],bytes32[][])", rawActions, _proofs));
```

Comment:
Why do we use `encodeWithSignature` here?

Reply:
If we just call the function directly, it will revert with the custom error immediately. I switched this to a low-level call so that the script doesn't instantly revert and instead we can decode the action index and output a more helpful error message.